tutorials/appstudio-onboarding/appstudio-onboarding.md

```diff
@@ -117,13 +121,17 @@ In this step, you will add the SAP Business Application Studio subscription to a
 
     !![legalterms](2020-03 AppStudio Terms__.jpg)
 
-6. A new tab opens and SAP Business Application Studio loads.
+6. A new tab opens and the welcome page for SAP Business Application Studio loads.
+
+    !![sapbusinessapplicationstudiowelcome](BAS_Welcome_.png)
```

Comment:
Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.